### ISSUE TITLE
[Bug] VoiceOver does not work for dropdown menu logout

### ISSUE THREAD

**Issue by thetif** [CMSgov/eAPD#3175]
**Description-**
VoiceOver can not navigate to the menu items in the dropdown menu on Firefox, Chrome, Edge - Safari time allowing.

**Type of Bug-**

- [X] Accessibility
- [ ] Functional
- [ ] Usability
- [ ] API
- [ ] Regression

**Steps to reproduce-**

1. Login as any user in Firefox
2. Turn on VoiceOver
3. Attempt to navigate to the logout button

**Role Affected-**
- [X] All
- [ ] Federal Admin
- [ ] State Admin
- [ ] State Staff
- [ ] State Contractor
- [ ] System Admin

**Browser-**
- [X] Firefox
- [x] Chrome
- [X] Edge
- [X] Safari
- [ ] IE

**Expected results-**
User should be able to access the logout button from the keyboard using VoiceOver

**Actual results-**
User can open the dropdown menu, but is unable to select anything in the dropdown.

**Related user story/bug-** (if applicable)
#3126

**Additional information-** (optional)
This is currently happening in staging so it is an existing issue.

**Logs or error output/Screenshot/Video-** (at least 1 of these)
https://images.zenhubusercontent.com/113234315/2174ba66-5032-4825-804e-39e7f966a69b/screen_recording_2021_06_01_at_2_00_29_pm.mov

**Comment by Sun-Mountain** [comment]
All found navigation bugs with screen readers have been captured in the [Accessibility in the Quality Process wiki page](https://github.com/CMSgov/eAPD/wiki/Accessibility-in-the-Quality-Process).

Signed off by @thetif
